_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,171 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Notifier can dispatch a variety of messages through various channels
+and varying formats alerting you to events in your environment.
+
+## Notification Types
+
+  * Inventory Summary: Count of the resources added in the latest inventory crawl.
+  * Violation Notifications: Individual violations that have been found from the latest scanner run. 
+  * Cloud SCC Findings: Violations converted to [Cloud SCC (Cloud Security Command Center)](https://cloud.google.com/security-command-center/) findings format, that's ingestable by Cloud SCC.
+  
+## Notification Channels
+
+  * Email
+  * Slack
+  * Cloud Storage
+
+## Notification Formats
+
+  * Human-readable data: CSV
+  * Structured data: JSON
+
+## Configuring Notifier
+
+### Inventory Summary
+
+This is a count of what resources have been crawled into inventory,
+and outputted to a Cloud Storage bucket.
+
+To configure how you want Notifier to send the Inventory Summary,
+follow the steps below
+
+1. Open `forseti-security/configs/server/forseti_conf_server.yaml`.
+
+1. Navigate to the `notifier` > `inventory` > `summary` section.
+
+```yaml
+notifier:
+    inventory:
+      summary:
+        enabled: true
+        data_format: csv
+        gcs_path: gs://path_to_foo_bucket
+```
+
+* `enabled`: `true` or `false`. Enables/disables the notification.
+* `data_format`: `csv` or `json`
+* `gcs_path`: The Cloud Storage bucket to upload the inventory summary.
+
+
+### Violation Notifications
+
+To configure how you want Notifier to send violation notifications,
+follow the steps below:
+
+1. Open `forseti-security/configs/server/forseti_conf_server.yaml`.
+1. Navigate to the `notifier` > `resources` section.
+
+On a per-resources basis, the following options are available. You can use
+any combination of notifiers for each resource.
+
+* `should_notify`: `true` or `false`.  Whether a violation for each resource
```

Comment:
Same as above, document this configuration variables in the same way.

Reply:
Done.